src/languageservice/serviceclient.ts

```diff
@@ -119,6 +126,14 @@ export default class SqlToolsServiceClient {
         this._client = client;
     }
 
+    private get resourceClient(): LanguageClient {
+        return this._resourceClient;
+    }
+
+    private set resourceClient(client: LanguageClient) {
+        this._resourceClient = client;
+    }
```

Comment:
No real point in having getters/setters if all you're doing is just modifying the private var directly